generator/README.md

```diff
@@ -0,0 +1,18 @@
+# SIG Doc builder
+
+This script will generate the following documentation files:
+
+```
+sig-*/README.md
+sig-list.md
+```
+
+Based off the `sigs.yaml` metadata file.
+
+## How to use
```

Comment:
I would request having more detailed documentation on how to use this tool (together with extended description).